os/include/stdlib.h

```diff
@@ -551,17 +551,22 @@ lldiv_t lldiv(long long numer, long long denom);
  */
 
 /**
- * @cond
- * @internal
+ * @ingroup STDLIB_LIBC
+ * @brief generates a unique temporary filename from template.
+ * @details @b #include <stdlib.h> \n
+ * @return always returns template.
+ * @since TizenRT v2.0 PRE
```

Comment:
There is no tc for mktemp and mkstemp. That means we can't add these comment.

Reply:
The tmpnam and tempnam tc's covers mktemp and mkstemp api's. Do we need to add new tc's for mktemp and mkstemp?

Reply:
Result is "not allowed".
Because internal called API is not tested all of cases. it is difficult to say "it covers internal called API" like this tc.

Reply:
Ok, will update with new tc's